docs/users_guide/ocean/test_groups/baroclinic_gyre.rst

```diff
@@ -0,0 +1,171 @@
+.. _baroclinic_gyre:
+
+baroclinic_gyre
+=======================
+
+The ``baroclinic_gyre`` test group implements variants of the
+baroclinic ocean gyre set-up from the 
+`MITgcm test case <https://mitgcm.readthedocs.io/en/latest/examples/baroclinic_gyre/baroclinic_gyre.html>`_.
```

Comment:
I think sphinx is getting confused by having this same link twice.  I would suggest putting the following at the bottom of the page:
```
.. _MITgcm test case: https://mitgcm.readthedocs.io/en/latest/examples/baroclinic_gyre/baroclinic_gyre.html
```
And then take the URL out of this and the following link:
```suggestion
`MITgcm test case`_.
```

Reply:
Thanks for the suggestion, I modified accordingly. The documentation built fine for me, and the link was correct.